themes/finna2/templates/RecordDriver/SolrAuthDefault/data-sources.phtml

```diff
@@ -6,7 +6,10 @@
     }
   ?>
   <?php if (!empty($url['url'])): ?>
-    <a href="<?=$this->escapeHtmlAttr($url['url'])?>"><?=$title?></a><?= $this->escapeHtml($url['subtitle']) ? (' ' . $this->escapeHtml($url['subtitle'])) : '' ?>
+    <a href="<?=$this->escapeHtmlAttr($url['url'])?>" target="_blank" aria-label="<?=$this->escapeHtmlAttr($url['title'])?> (<?=$this->transEscAttr('external_link')?>; <?=$this->transEscAttr('Open in a new window')?>)">
+      <i class="fa fa-external-link"></i> <?=$title?>
```

Comment:
Käytetään uusissa kohdissa icon-helperiä (tyyliin `<?=$this->icon('external-link')?>`)